python/ray/ml/train/integrations/lightgbm/lightgbm_trainer.py

```diff
@@ -1,14 +1,21 @@
 from typing import Optional, Dict, Any
+import os
 
-from ray.ml.trainer import GenDataset, Trainer
+from ray.ml.trainer import GenDataset
 from ray.ml.config import ScalingConfig, RunConfig
 from ray.ml.preprocessor import Preprocessor
 from ray.ml.checkpoint import Checkpoint
-from ray.util import PublicAPI
+from ray.ml.utils.gbdt_utils import GBDTTrainer, PARAMS_KEY, DMATRIX_PARAMS_KEY
```

Comment:
should we move these constants to ray.ml.constants?

Reply:
I am not sure, they are only relevant to XGB/LGBM, unlike other constants

Reply:
Ok we can leave it here for now, but I think it would make sense to prepend the variable names with GBDT_, just to be very explicit about which trainers these are meant for.